src/colorizer/shaders/colorize_RGBA8U.frag

```diff
@@ -84,6 +96,22 @@ vec4 getFloatFromTex(sampler2D tex, int index) {
   return texelFetch(tex, featurePos, 0);
 }
 
+uint getId(vec2 sUv) {
+  uint rawId = combineColor(texture(frame, sUv));
+  if (rawId == 0u) {
+    return BACKGROUND_ID;
+  }
+  uvec4 c = getUintFromTex(segIdToGlobalId, int(rawId - segIdOffset));
+  // Note: IDs are offset by `1` to reserve `0` for segmentations that don't
+  // have associated data. `1` MUST be subtracted from the ID when accessing
```

Comment:
Just curious: would it make sense to define -1 and 0 as constants rather than using literal values? Similar to `segIdOffset`

Reply:
Nevermind! Changed it to a constant.